Remove dApp Staking v2 code

bobo-k2 opened the pull request:
**Pull Request Summary**

Removed dApp staking v2 code and some other unused code. Also made adjustments to code which referenced removed code.

Things to do
- remove unused translation (tricky and time consuming one)
- integration tests

**Check list**

- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

gtg7784 commented on src/staking-v3/hooks/useDapps.ts:
```diff
@@ -99,6 +106,72 @@ export function useDapps() {
+
+      return true;
+    } catch (e) {
+      const error = e as unknown as AxiosError;
```
lets use `axios.isAxiosError` for error handling

https://github.com/axios/axios?tab=readme-ov-file#typescript

impelcrypto reviewed (approved):
Tested and it works. 
I think you can move the files from `staking-v3` (this can be done in this or the other PR)

impelcrypto commented:
@bobo-k2 I've resolved conflict in [this](https://github.com/AstarNetwork/astar-apps/pull/1283/commits/8e31b0a9fa80b50ae1f30e8aceb0fabd5300c2dd) commit.

bobo-k2 commented:
@impelcrypto, @gtg7784 thank you for comments. I resleoved them in https://github.com/AstarNetwork/astar-apps/pull/1283/commits/8e31b0a9fa80b50ae1f30e8aceb0fabd5300c2dd

Kahonnohak commented:
start checking now and found dApp news is missing 🫣
<img width="705" alt="Screenshot 2024-05-15 at 15 42 42" src="https://github.com/AstarNetwork/astar-apps/assets/77480847/d7b2831e-08ac-4766-a7af-9d4aa0f399a5">

Kahonnohak commented:
Could you remove data from dApp news section apart from the latest 5? Keep what we can see and remove all (only promotion ones)
<img width="1468" alt="Screenshot 2024-05-15 at 15 45 08" src="https://github.com/AstarNetwork/astar-apps/assets/77480847/addaa4a3-364a-42ab-af45-115bfe6791b1">

bobo-k2 commented:
> Tested and it works. I think you can move the files from `staking-v3` (this can be done in this or the other PR)

I will move in another PR

bobo-k2 commented:
> Could you remove data from dApp news section apart from the latest 5? Keep what we can see and remove all (only promotion ones) 

@Kahonnohak thank you for comments. I fixed missing ads area. I am not sure if I get comment regarding number of dApps correctly. I changed the code to show 5 promoted and 5 new dApps. Let me know if needs to be different.

Fixed in https://github.com/AstarNetwork/astar-apps/pull/1283/commits/db3a394a52b66fb7d094be1e3c151715fe62176c

gtg7784 reviewed (approved):
LGTM, thanks!

Kahonnohak reviewed (approved):
LGTM
